lib/sensors/INA_226.cpp

```diff
@@ -0,0 +1,79 @@
+#include "mbed.h"
+#include "INA_226.h"
+
+INA_226::INA_226(int i2c_frequency, PinName SDA_pinname, PinName SCK_pinname, u_int8_t sensor_address, float max_expected_current, float shunt_resistance)
+{
+    shunt_resistance_ = shunt_resistance;
+    max_expected_current_ = max_expected_current;
+    sensor_address_ = sensor_address << 1;
+
+    i2c_(SDA_pinname, SCK_pinname);
+    i2c_.frequency(i2c_frequency);
+}
+
+INA_226::~INA_226(){}
+
+//TODO: may need to adjust function to take sensor slave addresses as there many be many current sensors
+float Safety_Board::getCurrentData()
+{
+    //may need char16_t?
+    u_int8_t cmd[3] = {CURRENT_REGISTER};
+    
+    i2c_.write(sensor_address_, cmd, 1);
+    i2c_.read(sensor_address_, cmd, 2, false);
+    
+    u_int16_t current_data = (cmd[2] << 8 | cmd[1]);
+
+    return current_data;
+}
+
+
+float Safety_Board::getVoltageData()
+{
+    u_int8_t cmd[3] = {VOLTAGE_REGISTER};
+    
+    i2c_.write(sensor_address_, cmd, 1);
+    i2c_.read(sensor_address_, cmd, 2, false);
+    
+    u_int16_t voltage_data = (cmd[2] << 8 | cmd[1]);
+    

```

Comment:
you do still need the conversion ratio. "The LSB for the Bus Voltage Register (02h) is a fixed 1.25 mV/bit", top of pg 16 on data sheet